src/zvknhk.adoc

```diff
@@ -0,0 +1,435 @@
+
+[[zvknhk,zvknhk]]
+== "Zvknhk" Extension for Vector Keccak 
+
+This chapter describes the `Zvknhk`  Vector Cryptography extension, which adds vector instructions to compute the Keccak sponge function using in NIST SHA3 and numerous Post Quantum Cryptography (PQC) algorithms.
```

Comment:
The FIPS-202 spec formally defines the Keccak construction. Can we put a reference to the publication?